index.js

```diff
@@ -45,52 +50,85 @@ function getLatestData(caseData) {
 }
 
 async function getCovidData(channelId, query) {
-  query = isEmpty(query) ? 'all' : query;
-  if (query == 'all') {
+  console.log(query);
+  query = isEmpty(query) ? ['all'] : (Array.isArray(query) ? query : [query]);
+  if (query[0] == 'all') {
```

Comment:
```suggestion
  if (queryArray[0] === 'all') {
```

JS pretty much wants you to use `===` unless absolutely necessary nowadays. It prevents the fairly dumb typing system from stomping on itself.